Add info about copy paste grouped style categories (#10099)
qgis/QGIS-Documentation: docs/user_manual/introduction/general_tools.rst

   Right-click on the layer in the :guilabel:`Layers` panel to copy, paste, add
   or rename layer styles.

>    For both vector and raster layers, you can choose which style categories to copy/paste.

Reviewer: ```suggestion
   For both vector and raster layers, you can choose which style categories to copy/paste:
```

>    Choose :guilabel:`All Style Categories` or one of the available categories.
>    For vector layers, you can also select :guilabel:`All Symbology and Labeling Categories` option and :guilabel:`All Fields and Attribute Form Categories`
>    and for the raster layer you can choose :guilabel:`All Symbology and Labeling Categories` option.
>    Keep in mind, for the raster layer, the :guilabel:`Symbology` and :guilabel:`Rendering` categories are always copied together.

Reviewer: How about this? We use list items to provide visibility to each item, follow the GUI order (from the grouped ones to the single ones)
```suggestion

    * :guilabel:`All Style Categories`
    * :guilabel:`All Symbology and Labeling Categories` (including diagrams),
       and :guilabel:`All Fields and Attribute Form Categories` for vector layers
    * :guilabel:`All Symbology and Labeling Categories` option for raster layers
    * or one of the available categories.
    
    Keep in mind that, for the raster layer, the :guilabel:`Symbology`
    and :guilabel:`Rendering` categories are always copied together.
```
`or one of the available categories` could become "a specific/individual/single category"...
